### PR TITLE
Privacy policy

### PR DESCRIPTION
**Pull request by nickearnshaw**
Added footer with company information and link to privacy policy.
Rudimentary page with privacy policy (export from word with a bit of cleanup, forgive the horrible html)

### CODE REVIEW

File: _includes/privacy_policy_export.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">Eventbrite; </span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">Google.</span></li>
+</ol>
+</ol>
+<p><strong>Data that is received from publicly available third parties sources</strong></p>
+<p><span style="font-weight: 400;">9. We will receive Data about you from the following publicly available third party sources:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">Github; </span></li>
```
Comment (jukecraft) [inline _includes/privacy_policy_export.html]: do we need to be more specific and use proper company names here?

File: _includes/privacy_policy_export.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+<p><span style="font-weight: 400;">4. For purposes of the applicable Data Protection Laws, SOFTWARE CRAFT AND TESTING UK CIC t/a SoCraTes UK is the "data controller". This means that SOFTWARE CRAFT AND TESTING UK CIC t/a SoCraTes UK determines the purposes for which, and the manner in which, your Data is processed.</span></p>
+<p><strong>Data collected</strong></p>
+<p><span style="font-weight: 400;">5. We may collect the following Data, which includes personal Data, from you:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">name;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">contact Information such as email addresses and telephone numbers;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">demographic information such as postcode, preferences and interests;</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">financial information such as credit / debit card numbers;</span></li>
```
Comment (tomphp) [inline _includes/privacy_policy_export.html:76]: Do we ever collect financial information? It feels like all of that is handled elsewhere? (maybe this includes things likethings like invoice addresses?)

File: _includes/privacy_policy_export.html [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+<li style="font-weight: 400;"><span style="font-weight: 400;">Github; </span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">Twitter; </span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">Slack.</span></li>
+</ol>
+</ol>
+<p><strong>Data that is collected automatically</strong></p>
+<p><span style="font-weight: 400;">10. To the extent that you access the Website, we will collect your Data automatically, for example:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">we automatically collect some information about your visit to the Website. This information helps us to make improvements to Website content and navigation, and includes your IP address, the date, times and frequency with which you access the Website and the way you use and interact with its content.</span></li>
```
Comment (tomphp) [inline _includes/privacy_policy_export.html:122]: We currently don't do any of this ourselves. Maybe it needs to be there (or even explicit) that the services (i.e. github hosting) we use might.

File: _includes/privacy_policy_export.html [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+<li style="font-weight: 400;"><span style="font-weight: 400;">for other types of e-marketing, we are required to obtain your explicit consent; that is, you need to take positive and affirmative action when consenting by, for example, checking a tick box that we'll provide.</span></li>
+<li style="font-weight: 400;"><span style="font-weight: 400;">if you are not satisfied about our approach to marketing, you have the right to withdraw consent at any time. To find out how to withdraw your consent, see the section headed "Your rights" below.</span></li>
+</ol>
+</ol>
+<p><strong>Who we share Data with</strong></p>
+<p><span style="font-weight: 400;">14. We may share your Data with the following groups of people for the following reasons:</span></p>
+<ol>
+<ol style="list-style-type: lower-alpha;">
+<li style="font-weight: 400;"><span style="font-weight: 400;">our employees, agents and/or professional advisors - in order to facilitate the organisation and running of our events;</span></li>
```
Comment (tomphp) [inline _includes/privacy_policy_export.html]: I'm not sure this covers volunteers - do we need to either update this or decide how we control the access of data?

File: _includes/privacy_policy_export.html [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+<li style="font-weight: 400;"><strong>Right to data portability</strong><span style="font-weight: 400;"> - the right to request that we move, copy or transfer your Data.</span></li>
+<li style="font-weight: 400;"><strong>Right to object</strong><span style="font-weight: 400;"> - the right to object to our use of your Data including where we use it for our legitimate interests.</span></li>
+</ol>
+</ol>
+<p><span style="font-weight: 400;">21. To make enquiries, exercise any of your rights set out above, or withdraw your consent to the processing of your Data (where consent is our legal basis for processing your Data), please contact us via this e-mail address: privacy@socratesuk.org.</span></p>
+<p><span style="font-weight: 400;">22. If you are not satisfied with the way a complaint you make in relation to your Data is handled by us, you may be able to refer your complaint to the relevant data protection authority. For the UK, this is the Information Commissioner's Office (ICO). The ICO's contact details can be found on their website at https://ico.org.uk/.</span></p>
```
Comment (tomphp) [inline _includes/privacy_policy_export.html]: Perhaps make the URL a link as well?

File: _includes/privacy_policy_export.html [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+<li style="font-weight: 400;"><strong>Right to erase</strong><span style="font-weight: 400;"> - the right to request that we delete or remove your Data from our systems.</span></li>
+<li style="font-weight: 400;"><strong>Right to restrict our use of your Data</strong><span style="font-weight: 400;"> - the right to "block" us from using your Data or limit the way in which we can use it.</span></li>
+<li style="font-weight: 400;"><strong>Right to data portability</strong><span style="font-weight: 400;"> - the right to request that we move, copy or transfer your Data.</span></li>
+<li style="font-weight: 400;"><strong>Right to object</strong><span style="font-weight: 400;"> - the right to object to our use of your Data including where we use it for our legitimate interests.</span></li>
+</ol>
+</ol>
+<p><span style="font-weight: 400;">21. To make enquiries, exercise any of your rights set out above, or withdraw your consent to the processing of your Data (where consent is our legal basis for processing your Data), please contact us via this e-mail address: privacy@socratesuk.org.</span></p>
```
Comment (tomphp) [inline _includes/privacy_policy_export.html]: We should probably make this email address a link like the one in item 16.

**Review by tomphp — COMMENTED** [review]
I've added some thoughts - I'd be quite happy to merge it as is but just thought highlight the bits I'm not sure about.

File: _includes/privacy_policy_export.html (hunk 3 above)
Comment (nickearnshaw) [reply _includes/privacy_policy_export.html:122]: I think we just keep it as the platforms we are using do.

File: _includes/privacy_policy_export.html (hunk 2 above)
Comment (nickearnshaw) [reply _includes/privacy_policy_export.html:76]: We are more likely to now we have our own bank account. Currently evenbrite handles all the processing so it's more of a future proof statement.

File: _includes/privacy_policy_export.html (hunk 1 above)
Comment (nickearnshaw) [reply _includes/privacy_policy_export.html]: I think trading names are sufficient, but it's definitely better to provide full name so will update

File: _includes/privacy_policy_export.html (hunk 4 above)
Comment (nickearnshaw) [reply _includes/privacy_policy_export.html]: Adding a clause that includes volunteers

**Review by tomphp — APPROVED** [review]
Looks good to me 👍

**Pull request merged**